public/doc/openapi-3-v3.0.json

```diff
@@ -1914,6 +1914,15 @@
       "Source": {
         "type": "object",
         "properties": {
+          "app_creation_workflow": {
```

Comment:
Should this be a 3.x change instead of added to v3?

Reply:
Yes I like this idea. I will change to v3.1 in my other index PR and update this one with those changes.

Reply:
I'll rebase this PR onto master once #306 is merged.